### PR TITLE
IPU: support in place upgrade from RHEL9 to RHEL10

### PR DESCRIPTION
**Pull request by MiriamDeng**
Support in place upgrade test from RHEL9 to RHEL10
ID:2857

### CODE REVIEW

**Comment by MiriamDeng** [comment]
After dicussing the patch with yihyu and bfu, I've moved this patch to draft status. Will update it soon

**Comment by MiriamDeng** [comment]
It depends on internal kar patch #1379

**Comment by MiriamDeng** [comment]
@PaulYuuu could you please help to review this test case if you are free ? Thank you !

File: qemu/tests/cfg/in_place_upgrade.cfg [hunk 1]
```diff
@@ -63,7 +63,12 @@
             # com_ins_leapp = "yum install leapp-upgrade"
         - upstream_install:
             com_install = "no"
+            com_ins_leapp_9 = "dnf install leapp-upgrade-el9toel10 -y"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/in_place_upgrade.cfg:66]: Should not version related, the leapp should be a genreic case.

**Comment by MiriamDeng** [comment]
Updated @PaulYuuu can you give a hand to review it again ?

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**